misc/simulators/ringsim.py

```diff
@@ -130,7 +130,7 @@ def dump_usage(self, numfiles, avg_space_per_file):
         # used is actual per-server ciphertext
         usedpf = [1.0*u/numfiles for u in used]
         # usedpf is actual per-server-per-file ciphertext
-        #print "min/max usage: %s/%s" % (abbreviate_space(used[-1]),
+        #print("min/max usage: %s/%s" % (abbreviate_space(used[-1]),)
```

Comment:
This is also wrong.

Reply:
Fixed in 5a40bf4 and confirmed there aren't other cases of `,)` appearing at the end of a commented print line.